spec/models/step_spec.rb

```diff
@@ -38,4 +40,52 @@
       expect(step_item.errors).to have_key(:logic)
     end
   end
+  describe 'link reports' do
+    it 'should return nothing if there are no link reports yet' do
+      expect(step_item.broken_links).to be_nil
+    end
+
+    it 'should contain one item if there are link reports and at least one is broken' do
```

Comment:
Can you add a test that checks that `nil` is returned if a broken link report has been created, but there aren't any broken links.

You only need to create one link report to test this.